.github/workflows/e2e-test.yaml

```diff
@@ -73,29 +73,34 @@ jobs:
       - name: Set env
         run: |
           ARCHIVE_DIR=${{ github.workspace }}/images
+          REMOVER_REPO=remover
           ERASER_REPO=eraser
           MANAGER_REPO=manager
           COLLECTOR_REPO=collector
           TRIVY_SCANNER_REPO=scanner
 
+          REMOVER_TAG=test
           ERASER_TAG=test
           MANAGER_TAG=test
           COLLECTOR_TAG=test
           TRIVY_SCANNER_TAG=test
 
+          echo REMOVER_REPO=$REMOVER_REPO >> $GITHUB_ENV
           echo ERASER_REPO=$ERASER_REPO >> $GITHUB_ENV
           echo MANAGER_REPO=$MANAGER_REPO >> $GITHUB_ENV
           echo COLLECTOR_REPO=$COLLECTOR_REPO >> $GITHUB_ENV
           echo TRIVY_SCANNER_REPO=$TRIVY_SCANNER_REPO >> $GITHUB_ENV
 
-          echo ERASER_TAG=$ERASER_TAG >> $GITHUB_ENV
+          echo REMOVER_TAG=$REMOVER_TAG >> $GITHUB_ENV
           echo MANAGER_TAG=$MANAGER_TAG >> $GITHUB_ENV
+          echo ERASER_TAG=$ERASER_TAG >> $GITHUB_ENV
```

Comment:
Why have both `ERASER` and `REMOVER`?

Reply:
for the e2e tests, HelmDeployLatestEraserRelease() uses Eraser before we call UpgradeEraserHelm() with Remover